mlflow/store/db_migrations/versions/728d730b5ebd_add_registered_model_tags_table.py

```diff
@@ -22,7 +22,6 @@ def upgrade():
                     sa.Column('value', sa.String(length=5000)),
                     sa.Column('name', sa.String(length=256),
                               sa.ForeignKey('registered_models.name',
-                                            ondelete='cascade',
                                             onupdate='cascade'),
                               primary_key=True, nullable=False),
                     sa.PrimaryKeyConstraint('key', 'name', name='registered_model_tag_pk')
```

Comment:
@zhidongqu-db I think we're missing the backref on the `SqlRegisteredModel` table that we're defining in `models.py`, right?:

```
    registered_model = relationship('SqlRegisteredModel',
                                    backref=backref('registered_model_tags',
                                                    cascade='all'))
```

Reply:
Not an issue (see https://stackoverflow.com/questions/28108500/alembic-create-relationship-in-revision-file)